scoap3/modules/workflows/workflows/articles_upload.py

```diff
@@ -243,7 +251,7 @@ def build_files_data(obj, eng):
         ]
     elif method == 'scoap3_push':
         files = []
-        for document in obj.data.get('documents', ()):
+        for document in obj.data.pop('documents', ()):
```

Comment:
I am not sure if we want to drop the documents. These are just metadata of attached pdfs and doesn't contain any sensitive information

Reply:
You're right, it doesn't contain sensitive information, but I believe it's not needed either. It only contains the url for the files we are going to upload, so later this will be available through the `_files` field.
Also this in only present for the pushed articles.